app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -66,7 +86,7 @@ If you’re not a UK citizen, you are unlikely to get help funding your training
 
 ## Check your eligibility for a scholarship or bursary
 
-You may also be eligible for a scholarship or bursary, depending on the subject you're training to teach.
+You may also be eligible for a tax-free scholarship or bursary, depending on the subject you're training to teach.
 
 You do not have to pay a scholarship or bursary back. You can receive this as well as any tuition fee and maintenance loans.
 
```

Comment:
We link to the Find promo here at the end of this page which seems slightly odd - better for people to get to the adviser promo?